plugins/connection/httpapi.py

```diff
@@ -97,6 +97,18 @@
     default: true
     vars:
     - name: ansible_httpapi_use_proxy
+  ciphers:
+    description:
+      - SSL/TLS Ciphers to use for requests
+      - 'When a list is provided, all ciphers are joined in order with C(:)'
```

Comment:
I'm not sure why you removed the version_added, you just had the version of the wrong thing.
```suggestion
      - 'When a list is provided, all ciphers are joined in order with C(:)'
    version_added: 4.2.0
```